mutornadomon/collectors/ioloop_util.py

```diff
@@ -0,0 +1,18 @@
+import time
+
+
+class UtilizationCollector(object):
+
+    def __init__(self, monitor):
+        self.last_start_time = time.time()
+        self.monitor = monitor
+
+    def __enter__(self):
+        now = time.time()
+        self.last_start_time = now
+
+    def __exit__(self, *args, **kwargs):
+        duration = (time.time() - self.last_start_time)
+        self.last_start_time = time.time()
+        self.monitor.kv('callback_duration', duration)
+        self.monitor.count('accumulated_callback_duration', duration)
```

Comment:
Using `count` seems wrong here. `duration` is a float, `count` is meant for use with integers.
You can do the aggregation with your metrics collection mechanism (i.e. use `sumSeries` in graphite),
I don't see a need to implement aggregation here.